From aa4f75932a0ea85c350a05a05f90b4aa95d0d711 Mon Sep 17 00:00:00 2001
From: Kshitiz Varshney <kshitiz.varshney@nxp.com>
Date: Mon, 22 Aug 2022 11:09:27 +0200
Subject: [PATCH] OPTEE based Trusted key enablement for i.MX93

This commit modifies defconfig file for supporting OPTEE based trusted key which
is required for IMA/EVM.

Signed-off-by: Kshitiz Varshney <kshitiz.varshney@nxp.com>
---
 arch/arm64/configs/imx_v8_defconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index abea5f02f9a3..e85f7a39f9bd 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -1068,3 +1068,8 @@ CONFIG_TRUSTED_KEYS_CAAM=y
 
 CONFIG_SOC_IMX9=y
 CONFIG_ETHOSU=y
+
+# enable OPTEE based trusted key (for i.MX93)
+CONFIG_TRUSTED_KEYS_TEE=y
+CONFIG_TRUSTED_KEYS_CAAM=n
+
-- 
2.25.1

